### ISSUE TITLE
Weird verify message: Additional information: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.

### ISSUE THREAD

**Issue by MarjieAVolk** [StrangeLoopGames/EcoIssues#4545]
Got a repro on this "Additional information: Unexpected character encountered while parsing value: <. Path '', line 0, position 0."

This was the response from https://ecoauth.strangeloopgames.com/game/verifytokenv3?token=[...]
`"\n<div style="border:1px solid #990000;padding-left:20px;margin:0 0 10px 0;">\n\n<h4>A PHP Error was encountered</h4>\n\n<p>Severity: Notice</p>\n<p>Message:  Trying to get property of non-object</p>\n<p>Filename: controllers/Game.php</p>\n<p>Line Number: 825</p>\n\n\n\t<p>Backtrace:</p>\n\t\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\t\t<p style="margin-left:10px\">\n\t\t\tFile: /var/www/eco-account/application/modules/game/controllers/Game.php<br />\n\t\t\tLine: 825<br />\n\t\t\tFunction: _error_handler\t\t\t</p>\n\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\t\t<p style="margin-left:10px\">\n\t\t\tFile: /var/www/eco-account/index.php<br />\n\t\t\tLine: 292<br />\n\t\t\tFunction: require_once\t\t\t</p>\n\n\t\t\n\t\n\n</div>\n<div style="border:1px solid #990000;padding-left:20px;margin:0 0 10px 0;">\n\n<h4>A PHP Error was encountered</h4>\n\n<p>Severity: Notice</p>\n<p>Message:  Trying to get property of non-object</p>\n<p>Filename: controllers/Game.php</p>\n<p>Line Number: 825</p>\n\n\n\t<p>Backtrace:</p>\n\t\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\t\t<p style="margin-left:10px\">\n\t\t\tFile: /var/www/eco-account/application/modules/game/controllers/Game.php<br />\n\t\t\tLine: 825<br />\n\t\t\tFunction: _error_handler\t\t\t</p>\n\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\t\t<p style="margin-left:10px\">\n\t\t\tFile: /var/www/eco-account/index.php<br />\n\t\t\tLine: 292<br />\n\t\t\tFunction: require_once\t\t\t</p>\n\n\t\t\n\t\n\n</div>\n<div style="border:1px solid #990000;padding-left:20px;margin:0 0 10px 0;">\n\n<h4>A PHP Error was encountered</h4>\n\n<p>Severity: Notice</p>\n<p>Message:  Trying to get property of non-object</p>\n<p>Filename: controllers/Game.php</p>\n<p>Line Number: 825</p>\n\n\n\t<p>Backtrace:</p>\n\t\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\t\t<p style="margin-left:10px\">\n\t\t\tFile: /var/www/eco-account/application/modules/game/controllers/Game.php<br />\n\t\t\tLine: 825<br />\n\t\t\tFunction: _error_handler\t\t\t</p>\n\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\n\t\t\n\t\t\t<p style="margin-left:10px\">\n\t\t\tFile: /var/www/eco-account/index.php<br />\n\t\t\tLine: 292<br />\n\t\t\tFunction: require_once\t\t\t</p>\n\n\t\t\n\t\n\n</div>"`

**Comment by MarjieAVolk** [comment]
I just got this for myself when logging in via steam, and confirmed I do not have the account error which was causing this before.  Maybe this can help improve the error message.

**Comment by knexer** [comment]
Line 825 here is trying to unpack the steam response to get the error
description. So if the API request times out there is no response to unpack
and it returns this error.

**Comment by MarjieAVolk** [comment]
So you're saying this is because of the steam outage and otherwise wouldn't happen

**Comment by knexer** [comment]
Correct, that particular error that you posted anyways. Every error message
where the web server crashed would look exactly the same though, in terms
of 'error encountered whole parsing <' anyways.
